src/goto-instrument/cover_filter.h

```diff
@@ -85,7 +86,7 @@ class function_filterst
   /// \param identifier: function name
   /// \param goto_function: goto function
   bool operator()(
-    const irep_idt &identifier,
+    const symbolt &identifier,
```

Comment:
:pick: This is missing `override`

Reply:
Are you sure? The compiler does not like when I put `override` to this function.

Reply:
My mistake